### PR TITLE
Styling of login/signup and /performers

### PR DESCRIPTION
**Pull request by Andycutter**
## Description
- Apply styling to log in sign up popups and login button for FB and Google
- Added search field back in and made performers show up in rows of 6

performers_mock_search
login
artist_or_fan
signup_fan
signup_artist

### CODE REVIEW

File: app/views/devise/sessions/_form.html.haml [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+%body.modal-open
+    .modal-backdrop.fade.in
+        .modal-dialog{role: "document"}
+            .modal-content
+                .modal-body
+                    .mui-container-fluid
+                        .mui-row
+                            .mui-col-lg-8.mui-col-lg-offset-2.mui-col-md-12.modal
+                                %span.logo-venue
+                                %br/
```
Comment (tochman) [inline app/views/devise/sessions/_form.html.haml]: I’m not a great fan of using br. Can you set a top and bottom margin on the h4 below?

File: app/views/devise/sessions/_form.html.haml [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
+                                    .mui-textfield.mui-textfield--float-label
+                                        = f.password_field :password
+                                        = f.label :password
+                                    - if devise_mapping.rememberable?
+                                        = f.check_box :remember_me
+                                        = f.label :remember_me
+                                    %div
+                                    .form-group
+                                        = f.submit 'Login', class: 'form-submit'
+                                %div{style: 'margin-top: 20px;'}
```
Comment (tochman) [inline app/views/devise/sessions/_form.html.haml]: Why are the div below not indented?

File: app/views/partials/_header.html.haml [hunk 3]
```diff
@@ -23,7 +23,7 @@
                             %li.buttoned
                                 = link_to 'Log out', destroy_user_session_path, method: :delete
                         - else 
-                            %li
+                            %li.buttoned
```
Comment (tochman) [inline app/views/partials/_header.html.haml]: Why do you use buttonend on both li elements.  Shouldn’t only the last one have that class?

Comment (Andycutter) [reply app/views/partials/_header.html.haml]: If .buttoned is not applied then `Log in` will not have the yellow border on it.

Comment (Andycutter) [reply app/views/partials/_header.html.haml]: Oh I see now, that `Log in` is not supposed to have a yellow border according to the layout-pdf

File: app/views/devise/sessions/_form.html.haml (hunk 2 above)
Comment (Andycutter) [reply app/views/devise/sessions/_form.html.haml]: For no reason, it is now indented

**Pull request merged**